app/models/course/experience_points/disbursement.rb

```diff
@@ -79,15 +79,19 @@ def valid_points_record_attributes?(attibutes)
   #
   # @return [Array<CourseUser>] The list of potential students awardees
   def filtered_students
-    group_id ? students_from_group(group_id) : course.course_users.student.with_approved_state
```

Comment:
@kxmbrian Why not apply the scope directly to `filtered_students` in `#experience_points_records` ? so that we don't duplicate the scope twice.